bricksrc/equipment.py

```diff
@@ -558,6 +565,42 @@
             },
         },
     },
+    "Heat_Pump": {
```

Comment:
@gtfierro How could we model a heat pump which is a split system? These definitions work well for a packaged unit, such as the illustrated "Air_Source_Heat_Pump":
![image](https://github.com/BrickSchema/Brick/assets/33592545/5c142a31-1fd2-4c68-9145-86a811795055)

However, a split system would have both a Condensing Unit and an FCU. This, too, is a Heat Pump:
![image](https://github.com/BrickSchema/Brick/assets/33592545/1362a4d4-e9d9-4ca9-a89f-a1228df77c80)

Reply:
In some sense, a split system is more of a "Heat Pump System", where as a packaged unit is a "Heat Pump Equipment"

Reply:
We do have "Heat_Pump_Condensing_Unit", perhaps "Heat_Pump" class could be modified to "Heat_Pump_Packaged_Unit" for clarification.

Reply:
Is the existing `Condenser` not adequate? Perhaps we need a `Condenser_Unit` class and a corresponding `Evaporator_Unit` class.

Reply:
@MatthewSteen In the split-system example above, this would be achieved by using the [Heat_Pump_Condensing_Unit](https://github.com/mapped/Brick/blob/8e73f532db499395462747edf05fbd209895f378/bricksrc/equipment.py#L771C14-L771C39) > feeds > [ FCU](https://github.com/mapped/Brick/blob/8e73f532db499395462747edf05fbd209895f378/bricksrc/equipment.py#L388)

In this case, do you agree that [Heat_Pump](https://github.com/mapped/Brick/blob/8e73f532db499395462747edf05fbd209895f378/bricksrc/equipment.py#L672) should be "Packaged Heat Pump"?

https://www.trane.com/residential/en/products/packaged-systems/heat-pump-packaged-systems/

Reply:
Renamed "Heat Pump" to "Packaged_Heat_Pump" and added air/water/ground source to all condensing unit classes. In doing so, we can represent the same level of detail for both Packaged and Split System variations.
https://github.com/BrickSchema/Brick/pull/590/commits/cbcc5d47c49d9110478339c72ea59c3630ad008a